### PR TITLE
fix(openapi): the manual review action needs to be added with scopeID props

### PR DESCRIPTION
**Pull request by kakj-go**
#### What this PR does / why we need it:
The project pipeline is special, and the manual review action needs to be added with scopeID props

#### ChangeLog

| Language | Changelog |
| --------- | ------------ |
| 🇺🇸 English |     the manual review action needs to be added with scopeID props       |
| 🇨🇳 中文    |      人工审核 action 需要额外加上 scopeID props        |

### CODE REVIEW

File: modules/openapi/component-protocol/scenarios/action/components/actionForm/render.go [hunk 1]
```diff
@@ -330,12 +330,32 @@ func registerActionTypeRender() {
 				return nil
 			}
 
-			if c.Props != nil {
+			defer func() {
+				if c.Props == nil {
```
Comment (Effet) [inline modules/openapi/component-protocol/scenarios/action/components/actionForm/render.go:334]: encapsulate a func like `getScopeId()`

**Review by Effet — DISMISSED** [review]

File: modules/openapi/component-protocol/scenarios/action/components/actionForm/render.go (hunk 1 above)
Comment (Effet) [reply modules/openapi/component-protocol/scenarios/action/components/actionForm/render.go:334]: and add some unit test.

File: modules/openapi/component-protocol/scenarios/action/components/actionForm/render.go [hunk 2]
```diff
@@ -330,12 +330,32 @@ func registerActionTypeRender() {
...
+
+				for _, field := range fields {
+					if field.Component != "memberSelector" {
+						continue
+					}
+					props, ok := field.ComponentProps.(map[string]interface{})
+					if !ok {
+						continue
+					}
+					props["scopeId"] = bdl.InParams["appId"]
```
Comment (sfwn) [inline modules/openapi/component-protocol/scenarios/action/components/actionForm/render.go:356]: why change `scopeID` to `scopeId` ?

**Review by sfwn — CHANGES_REQUESTED** [review]

**Comment by sfwn** [comment]
you should add screenshots to show what this pr does.
one for before-fix, and another for fixed.

File: modules/openapi/component-protocol/scenarios/action/components/actionForm/render.go (hunk 2 above)
Comment (kakj-go) [reply modules/openapi/component-protocol/scenarios/action/components/actionForm/render.go:356]: In order to meet the requirements of front-end componentization

**Comment by kakj-go** [comment]
> you should add screenshots to show what this pr does. one for before-fix, and another for fixed.

If the memberSelector component in props does not add scopeId, it will not be able to get the user list of the application in the project pipeline

image

After reading the appId in inparams and setting it to the fields of the memberSelector component, the project pipeline can use the manual-review action without any problem.

image

**Review by sfwn — APPROVED** [review]

**Pull request merged**